backend/handler/igdb_handler.py

```diff
@@ -9,204 +9,270 @@
 from logger.logger import log
 
 
-class IGDBHandler():
-
+class IGDBHandler:
     def __init__(self) -> None:
-        base_url: str = 'https://api.igdb.com/v4'
-        self.platform_url: str = f'{base_url}/platforms/'
-        self.games_url: str = f'{base_url}/games/'
-        self.covers_url: str = f'{base_url}/covers/'
-        self.screenshots_url: str = f'{base_url}/screenshots/'
-        self.twitch_auth: TwitchAuth = TwitchAuth()
+        self.platform_url = "https://api.igdb.com/v4/platforms/"
+        self.games_url = "https://api.igdb.com/v4/games/"
+        self.covers_url = "https://api.igdb.com/v4/covers/"
+        self.screenshots_url = "https://api.igdb.com/v4/screenshots/"
+        self.twitch_auth = TwitchAuth()
         self.headers = {
-            'Client-ID': self.twitch_auth.client_id,
-            'Authorization': f'Bearer {self.twitch_auth.get_oauth_token()}',
-            'Accept': 'application/json'
+            "Client-ID": self.twitch_auth.client_id,
+            "Authorization": f"Bearer {self.twitch_auth.get_oauth_token()}",
+            "Accept": "application/json",
         }
 
-
     def check_twitch_token(func) -> tuple:
         @functools.wraps(func)
         def wrapper(*args):
-            args[0].headers['Authorization'] = f'Bearer {args[0].twitch_auth.get_oauth_token()}'
+            args[0].headers[
+                "Authorization"
+            ] = f"Bearer {args[0].twitch_auth.get_oauth_token()}"
             return func(*args)
-        return wrapper
 
+        return wrapper
 
-    def _search_rom(self, search_term: str, p_igdb_id: int, category: int = None) -> dict:
+    def _search_rom(
+        self, search_term: str, p_igdb_id: int, category: int = None
+    ) -> dict:
         category_filter: str = f"& category={category}" if category else ""
         try:
-            return requests.post(self.games_url,
-                                 headers=self.headers,
-                                 data=f"""search \"{search_term}\";
-                                      fields id, slug, name, summary, screenshots;
-                                      where platforms=[{p_igdb_id}] {category_filter};""",
-                                 timeout=120).json()[0]
+            return requests.post(
+                self.games_url,
+                headers=self.headers,
+                data=f"""search \"{search_term}\";
+                    fields id, slug, name, summary, screenshots;
+                    where platforms=[{p_igdb_id}] {category_filter};
+                """,
+                timeout=120,
+            ).json()[0]
         except IndexError:
             return {}
 
+    def _search_rom(
```

Comment:
is this `_search_rom` duplicated method maybe a mistake from the previous merge?

Reply:
yep, will fix in an upcoming commit